public/kibi_timeline.js

```diff
@@ -221,9 +304,35 @@ define(function (require) {
           initTimeline();
           if ($scope.groups) {
             initGroups();
-            // do not use newValue as it does not have searchSource as we filtered it out
-            _.each($scope.groups, initSingleGroup);
-            courier.fetch();
+
+            var indexeIds = _.map($scope.groups, (g) => {
+              return g.searchSource.get('index').id;
+            });
+
+            var fieldIds = [];
+            _.each($scope.groups, (g) => {
```

Comment:
You could iterate over $scope.groups only once to populate both fieldIds and indexIds .